src/code/code_800E6840.c

```diff
@@ -1,13 +1,13 @@
 #include "global.h"
 
-void Audio_InvalDCache(void* buf, s32 size) {
+void Audio_InvalDCache(void* buf, ssize_t size) {
     OSIntMask prevMask = osSetIntMask(1);
 
     osInvalDCache(buf, size);
     osSetIntMask(prevMask);
 }
 
-void Audio_WritebackDCache(void* buf, s32 size) {
+void Audio_WritebackDCache(void* buf, ssize_t size) {
     OSIntMask prevMask = osSetIntMask(1);
```

Comment:
This came from MM, but also applies here. These sizes from these two functions can be `u32` -> `size_t` instead of signed. You'll need to change to `size_t` in the function header file too though.